src/main/scala/beam/calibration/utils/DeleteSuggestion.scala

```diff
@@ -31,55 +31,44 @@ object DeleteSuggestion extends LazyLogging {
   def deleteSuggestions(experimentId: String, suggestions: List[String]) = {
 
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
+      case Some(_experiment) =>
         suggestions.foreach { suggestionId =>
-          {
-
-            _experiment.suggestions().delete(suggestionId).call()
-          }
+          _experiment.suggestions().delete(suggestionId).call()
         }
-      }
-      case None => {
+      case None =>
         logger.info(s"Experiment with id $experimentId not found")
-      }
     }
   }
 
   def listSuggestions(experimentId: String) = {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
-
-        if (_experiment.suggestions().list().call().getData.size() > 0)
-          _experiment.suggestions().list().call().getData.forEach { d =>
-            println(d)
-          } else
+      case Some(_experiment) =>
+        val data = _experiment.suggestions().list().call().getData
+        if (!data.isEmpty) {
+          data.forEach(println)
+        } else {
           logger.info(s"Experiement with id $experimentId has no suggestion")
-      }
-      case None => {
+        }
+      case None =>
         logger.info(s"Experiment with id $experimentId not found")
-      }
     }
   }
 
-  def deleteAllOpenSuggestions(experimentId: String) = {
-
+  def deleteAllOpenSuggestions(experimentId: String): Unit = {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
-
-        if (_experiment.suggestions().list().call().getData.size() > 0)
+      case Some(_experiment) =>
+        if (!_experiment.suggestions().list().call().getData.isEmpty) {
```

Comment:
Can you do the same cleanup here? Store the redundant `getData` into a top level variable?